lib/ansible/plugins/action/reboot.py

```diff
@@ -310,11 +310,15 @@ def do_until_success_or_timeout(self, action, reboot_timeout, action_desc, distr
                         error = to_text(e).splitlines()[-1]
                     except IndexError as e:
                         error = to_text(e)
-                    display.debug("{action}: {desc} fail '{err}', retrying in {sleep:.4} seconds...".format(
+                    msg = "{action}: {desc} fail '{err}', retrying in {sleep:.4} seconds...".format(
```

Comment:
Since action plugins run on the controller side, should we just use an f-string here?

Reply:
Done.